### PR TITLE
[zh] Sync #14146 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14146 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 1]
```diff
@@ -226,6 +226,24 @@ $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
 $ kubectl delete destinationrule egressgateway-for-wikipedia
 {{< /text >}}
 
+## 任意域的 wildcard 配置  {#wildcard-configuration-for-arbitrary-domains}
+
+上一节中的配置是有效的，因为所有 `*.wikipedia.org` 站点都可能由任何一个 `wikipedia.org` 服务器提供服务。
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:231]:
```suggestion
上一节中的配置之所以有效，是因为所有 `*.wikipedia.org` 站点都可能由任何一个 `wikipedia.org` 服务器提供服务。
```

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 2]
```diff
@@ -226,6 +226,24 @@ $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
...
+然而，实际情况并非总是如此。例如，您可能想要配置出口控制以访问更通用的 wildcard 域，
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:232]:
```suggestion
然而，实际情况并非总是如此。例如，您可能想要配置出口控制以访问更通用的通配符域名，
```

Comment (hanxiaop) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:232]: 我记得之前有个文字 wildcard 是翻译的？

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 3]
```diff
@@ -226,6 +226,24 @@ $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
...
+例如 `*.com` 或 `*.org`。配置任意 wildcard 域的流量给 Istio 网关带来了挑战；
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:233]:
```suggestion
例如 `*.com` 或 `*.org`。为任意通配符域名配置流量给 Istio 网关带来了挑战；
```

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 4]
```diff
@@ -226,6 +226,24 @@ $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
...
+Istio 网关只能将流量路由配置到预定义的主机、预定义的 IP 地址或请求的原始目标 IP 地址。
+
+在上一节中，您配置了虚拟服务用于将流量定向预定义主机 `www.wikipedia.org`。
+然而，在一般情况下，您不知道可以为请求中收到的任意主机提供服务的主机或 IP 地址，
+这使得请求的原始目标地址成为路由请求的唯一值。不幸的是，当使用出口网关时，
+由于原始请求被重定向到网关，因此请求的原始目标地址丢失，导致目标 IP 地址变成网关的 IP 地址。
+
+虽然不像依赖 Istio 实现细节那么简单且有些脆弱，但您可以使用
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:241]:
```suggestion
尽管这样做并不简单且有些脆弱，因为它依赖于 Istio 实现细节，但您可以使用
```

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:232]: 是的，不过这篇文档中的 wildcard 都没有翻译，所以我保持了这个风格。

Comment (hanxiaop) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:232]: 明白了

Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:232]: 那就暂时保持，稍后有空统一优化一波。

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: Done

**Review by windsonsea — APPROVED** [review]
Thanks. This term `wildcard` can be optimized in a separate PR later.

**Pull request merged**